pages/forms/registration/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>We Life</title>
+  </head>
+  <body>
+    <header>
+      <title>Registration Form</title>
+    </header>
+    <h1>Registration Form</h1>
+    <p>Please fill out this form with the required information</p>
+    <form method="post">
+      <fieldset>
+        <label for="name">Enter Your First Name:</label>
+        <input id="name" name="name" type="text" required />
+        <br>
```

Comment:
The correct way to use `br` is `<br />` since it is a self-closing tag element. 
Also, it should not be used to increase space between elements. Remove it from any place where it is not needed. 

You can read the reason for this [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns).